var/spack/repos/builtin/packages/openbabel/package.py

```diff
@@ -12,7 +12,7 @@ class Openbabel(CMakePackage):
     search, convert, analyze, or store data from molecular modeling, chemistry,
     solid-state materials, biochemistry, or related areas."""
 
-    homepage = "https://openbabel.org/wiki/Main_Page"
+    homepage = "https://openbabel.org/docs/index.html"
```

Comment:
```suggestion
    homepage = "https://openbabel.org/index.html"
```
Nice to see the new documentation live. I think the link above is better as `homepage`.